src/messaging/controllers/messaging.controller.ts

```
@@ -0,0 +1,90 @@
+    description: "User not found",
+  })
+  @ApiResponse({
+    status: HttpStatus.UNAUTHORIZED,
+    description: "User not authenticated",
+  })
+  async getConversation(@Request() req, @Query() params: GetMessagesDto) {
+    const userId = req.user.id
+    return this.messagingService.getConversation(userId, params)
+  }
+
+  @Patch("mark-read/:userId")
```

ESLint

[error] Replace `"mark-read/:userId"` with `'mark-read/:userId'`

(prettier/prettier)